README.md

```diff
@@ -447,6 +448,29 @@ Approve a declarative migration plan.
 * `link` - The URL of the plan in [Atlas Registry](https://atlasgo.io/registry).
 * `status` - The status of the plan. For example, `PENDING` or `APPROVED`.
 
+### `ariga/atlas-action/agent`
+
+Run an [Atlas Agent](https://atlasgo.io/cloud/agents) that execute cloud jobs on your database.
```

Comment:
This sentance is not clear enough about what the action is doing, how it works etc.